apps/web/app/(navigation)/layout.tsx

```diff
@@ -20,20 +26,70 @@ const Main = styled.main`
   padding-top: 68px;
 `;
 
-const Content = styled.div`
+const ContentUser = styled.div`
   flex-grow: 1;
   width: 100%;
-  padding-left: 94px;
+  padding-left: 25px;
+  ${device.tablet} {
+    padding-left: 94px;
+  }
+`;
+
+const ContentNoUser = styled.div`
+  flex-grow: 1;
+  width: 100%;
+  padding-left: 25px;
+`;
+
+const SideNavMobile = styled.div`
+  ${device.tablet} {
+    display: none;
+  }
+`;
+
+const SideNavDesktop = styled.div`
+  @media (max-width: 767px) {
+    display: none;
+  }
 `;
```

Comment:
why hardcoding it? if `tablet` size in media queries would change, then you have to remember to change this hardcoded value

```suggestion
const SideNavDesktop = styled.div`
  display: none;
  ${device.tablet} {
    display: block;
  }
`;
```

Reply:
done, thanks a lot for the review 🚀